docs/platforms/javascript/common/usage/index__v7.119.0.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Capturing Errors and Events
+description: "Learn how to use the SDK to manually capture errors and other events."
+sidebar_order: 4
+---
+!!Versioned index file!!
+
+
```

Comment:
There are two empty lines :) (ignore if this is on purpose)
```suggestion
```

Reply:
yeah that's just a demo file for testing, I'll remove these before merging 👍